src/pages/StatusPage.vue

```diff
@@ -272,11 +330,19 @@
                     <div v-if="allMonitorList.length > 0 && loadedData">
                         <label>{{ $t("Add a monitor") }}:</label>
                         <select v-model="selectedMonitor" class="form-control">
-                            <option v-for="monitor in allMonitorList" :key="monitor.id" :value="monitor">{{ monitor.name }}</option>
+                            <option v-for="monitor in allMonitorList" :key="monitor.id" :value="monitor">
+                                {{
+                                    monitor.name
+                                }}
+                                <span v-if="monitor.tags.length > 0">
+                                    [Tags: {{ monitor.tags.map(tag => tag.name).join(" ,") }} ]
+                                </span>
+                            </option>
```

Comment:
Can you drop a screenshot of how does this change make on the UI?

Reply:
sure, it looks like this
<img width="1728" alt="uptime-kuma-screenshot" src="https://user-images.githubusercontent.com/44889843/218374033-547b235e-b769-453e-a21f-1ef6fc3cb37b.png">

Reply:
Thank you, I think that's all we need for the functionality. 

Just a minor suggestion for the UI, do you think it would look better if we could display the tags as labels with color like this? We can get the color from the `tag.color` attribute.

![Screen Shot 2023-02-13 at 13 03 25](https://user-images.githubusercontent.com/19431276/218382660-32e0d801-2ec8-4c04-aac3-7c521eb35d0c.png)

Reply:
In that case, i think it is better if we implement `vue-multiselect` as the component instead of using a plain `select`. because a plain `select` does not render colors consistently. For example, In the above screenshot (mac os), you can see that the color will probably not be visible since it is using the styles provided by the host operating system (mac os in this case)

Reply:
ping @khanh96le

Reply:
Please validate the above idea @khanh96le so that I can know if I can start to work on refactoring the PR. Thanks !

Reply:
Hi @titanventura,

I am sorry for the late response.

I really appreciate the idea you proposed, it looks great to me! When you have a moment, can you share another screenshot of the updated version? I'm excited to see how it looks and I'm sure everyone else is too.

Thank you so much for your hard work and contributions!

Reply:
Hi @khanh96le, no probs. thank you so much for taking the time to analyse and suggest. I have updated the select monitor component in edit status page. It uses `vue-multiselect` and shows tags with name and value (if present) with the background color of the tag. It looks like this. 

![image](https://user-images.githubusercontent.com/44889843/219873360-e1ac5168-936d-45ed-a802-94b853e0ca0f.png)

Please do review and suggest changes if any. Thanks !

Reply:
BTW, the e2e tests seem to be flaky. It [dddc239](https://github.com/louislam/uptime-kuma/pull/2752/commits/dddc2395005270c6846b2475ed6808cec896dc44) passed but [388da6f](https://github.com/louislam/uptime-kuma/pull/2752/commits/388da6fc52ead03a9e326c03b81fa268dd35b318) failed